src/item_contents.cpp

```diff
@@ -2125,6 +2125,16 @@ void item_contents::process( map &here, Character *carrier, const tripoint &pos,
     }
 }
 
+bool item_contents::leak( map &here, Character *carrier, const tripoint &pos, item_pocket *pocke )
+{
+    for( item_pocket &pocket : contents ) {
+        if( pocket.is_type( item_pocket::pocket_type::CONTAINER ) ) {
+            pocket.leak( here, carrier, pos, pocke );
+        }
+    }
+    return false;
+}
```

Comment:
This function returns false in any case, right? If so, do we need this as a bool?

Reply:
Yeah, the return value is does not matter in the current process. I consider changing to void.

Reply:
Function changed from bool to void.